addons/mail/models/mail_template.py

```diff
@@ -623,7 +623,9 @@ def send_mail(self, res_id, force_send=False, raise_exception=False, email_value
             attachment_ids.append((4, Attachment.create(attachment_data).id))
         if attachment_ids:
             mail.write({'attachment_ids': attachment_ids})
-
-        if force_send:
-            mail.send(raise_exception=raise_exception)
+        try:
+            if force_send:
+                mail.send(raise_exception=raise_exception)
+        except:
+            raise UserError(_("Check outgoing mail server, is it successfully configured or not."))
```

Comment:
Mix of imperative and interogative is weird. I'd just say there was an error and what they can do to try to fix it. 
"We could not send your message. Please ensure your outgoing mail server is configured properly."